pulpcore/pulpcore/app/models/repository.py

```diff
@@ -247,6 +247,20 @@ def content(self):
         )
         return Content.objects.filter(version_memberships__in=relationships)
 
+    @property
+    def content_by_type(self):
```

Comment:
I'm not actually sure this function is necessary or even particularly useful, I may remove it before merging.  It seemed like an obvious thing to have at first but I realized that I can't think of anywhere it would actually be used.

Reply:
Should the ```added()``` and ```removed()``` methods be made properties, to match ```content```?  Or should we make ```content``` a method?  The latter would provide maximal flexibility in terms of being able to add functionality later on in the form of kwargs.